tox.ini

```diff
@@ -2,6 +2,7 @@
 envlist =
     {py27,py34,py35,py36}-django-{18,19,110,111}
     {py34,py35,py36}-django-{20}
+    {py35,py36,p37}-django-{21}
```

Comment:
Note that [python 3.4 support was dropped for 2.1](https://docs.djangoproject.com/en/2.1/releases/2.1/).